ARF Trusted Lists and Trust Infrastructure harmonizations (#988)
italia/eidas-it-wallet-docs: docs/en/defined-terms.rst

Defined Terms and Acronyms
==========================

This section aligns the IT-Wallet System's terminology with the definitions provided in ARF 2.7.3 (see `ARF Annex 1 <https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/main/docs/annexes/annex-1/annex-1-definitions.md>`_). For each term, the IT-Wallet definition is compared and mapped to the ARF definition, with notes on any differences or clarifications.

> The terms *User*, *Trust Service*, *Trust Model*, *eIDAS Trusted List*, *Trust Framework*, *Attribute*, *Electronic Attestations of Attributes Provider* or *Trust Service Provider (TSP)*, *Person Identification Data (PID)*, *Revocation List*, *Qualified Electronic Attestations of Attributes Provider* or *Qualified Trust Service Provider (QTSP)*, *Electronic Attestation of Attributes (EAA)*, are defined in the `EIDAS-ARF`_.

Reviewer: ```suggestion
The terms *User*, *Trust Service*, *Trust Model*, *Trusted List*, *Trust Framework*, *Attribute*, *Electronic Attestations of Attributes Provider* or *Trust Service Provider (TSP)*, *Person Identification Data (PID)*, *Revocation List*, *Qualified Electronic Attestations of Attributes Provider* or *Qualified Trust Service Provider (QTSP)*, *Electronic Attestation of Attributes (EAA)*, are defined in the `EIDAS-ARF`_.
```

**eIDAS Trusted List** is not a defined term in ARF

    **Issuer**
    **Attestation Provider**
      Organizational Entity providing Digital Credentials to Users (may be PID Provider or (Q)EAA Provider).
      Aligned with ARF 2.7.3; IT-Wallet merges PID and (Q)EAA Providers under this term.

    **Critical Assets**
      Assets (e.g., cryptographic keys) whose loss would seriously impact the Wallet Unit.

>       Aligned with ARF 2.5.

Reviewer: ```suggestion
      Aligned with ARF 2.7.3.
```

      Aligned with ARF 2.7.3.

    **Trust**
      Confidence in the security, reliability, and integrity of entities and their actions.
      Not present in ARF 2.7.3.

    **Trust Attestation**
      Electronic attestation of compliance with the regulatory framework, cryptographically verifiable.
      Not present in ARF 2.7.3.

    **Trust Evaluation**
      Process of verifying trustworthiness of registered Organizational Entities.
      Not present in ARF 2.7.3.

    **Trust Framework**
      Legally enforceable set of rules and agreements for a multi-party system.
      Not present in ARF 2.7.3.

    **Trust Layer**
      Architectural component enabling participants to establish trust.
      Not present in ARF 2.7.3.

    **Trust Model**
      Collection of rules ensuring legitimacy of components/entities in the IT-Wallet ecosystem.
      Not present in ARF 2.7.3.

    **Trust Relationship**
      Reliable relationship between Organizational Entities after Trust Evaluation.
      Not present in ARF 2.7.3.

    **Access Certificate**
      Certificate authenticating and validating the (Wallet-) Relying Party.

>       Aligned with ARF 2.7.3

Author: done in b09759adbba3e406cf2214e07269e59ee33d58e4

      Certificate authenticating and validating the (Wallet-) Relying Party.

>       Aligned with ARF 2.5.

Reviewer: ```suggestion
      Aligned with ARF 2.7.3
```

    **Access Certificate Authority**
    **Access CA**
      Certificate Authority responsible for issuing Access Certificates to PID Providers, Attestation Providers, and Relying Parties for secure interactions within the IT-Wallet ecosystem.

>       Aligned with ARF 2.5; terminology specialized for IT-Wallet.

Reviewer: ```suggestion
      Aligned with ARF 2.7.3; terminology specialized for IT-Wallet.
```

    **Registration Certificate**
      Data object indicating the attributes the Relying Party has registered to request from Users.

>       Aligned with ARF 2.5.

Reviewer: ```suggestion
      Aligned with ARF 2.7.3.
```

    **Registration Certificate Provider**
    **Provider of Registration Certificates**
    **Reg. Cert. Provider**
      Organizational Entity responsible for issuing Registration Certificates that describe the registration status and entitlements of PID Providers, Attestation Providers, and Relying Parties.

>       Aligned with ARF 2.5; terminology specialized for IT-Wallet.

Reviewer: ```suggestion
      Aligned with ARF 2.7.3; terminology specialized for IT-Wallet.
```

    **Federation Registry**
      Registry component of the IT-Wallet Registry Infrastructure that maintains the authoritative list of trusted entities participating in the federation, including their federation metadata, endpoints, and public keys.

>       Not present in ARF 2.5; specific to IT-Wallet.

Reviewer: ```suggestion
      Not present in ARF 2.7.3; specific to IT-Wallet.
```

    **Authentic Source Registry**
    **AS Registry**
      Registry containing Authentic Sources, their declared capabilities, and available claims, used by Credential Issuers to discover and integrate authoritative data providers.

>       Not present in ARF 2.5; specific to IT-Wallet.

Reviewer: ```suggestion
      Not present in ARF 2.7.3; specific to IT-Wallet.
```

    **Member State Trusted List Provider**
    **MS TLP**

>       Organizational Entity designated by a Member State to compile, sign, and publish national eIDAS Trusted Lists (e.g., QTSP Trusted Lists, EAA Provider Trusted Lists) and to notify their locations to the European Commission.

Reviewer: ```suggestion
      Organizational Entity designated by a Member State to compile, sign, and publish national Trusted Lists (e.g., QTSP Trusted Lists, EAA Provider Trusted Lists) and to notify their locations to the European Commission.
```

      Terminology specialized for IT-Wallet.

    **Certificate Signing Request**
      Request sent to a CA containing the public key and identifying information for a digital certificate.

>       Not present in ARF 2.5.

Reviewer: ```suggestion
      Not present in ARF 2.7.3.
```

    **eIDAS Trusted List**

>       Repository of information about authoritative entities and their status used in the eIDAS trust services framework (e.g., QTSP Trusted Lists, EAA Provider Trusted Lists).

Author: ```suggestion
      Repository of information about authoritative entities and their status used in the eIDAS trust services framework (e.g., PID Provider Trusted Lists, Wallet Provider Trusted Lists, List of Trusted List pointing to Member States QTSP Trusted Lists and Member States EAA Provider Trusted Lists, Access Certificates CA and Registristration Certificates Provider Trusted Lists).
```

>       Aligned with ARF 2.5.

Reviewer: ```suggestion
      Aligned with ARF 2.7.3.
```
